azerothcore/azerothcore-wotlk
# fix(DB/quest): Sets Apothecary Dithers as quest start/end for "The Ke…

…y to Scholomance"



## Changes Proposed:
- Sets [Apothecary Dithers](https://tbc.wowhead.com/npc=11057/apothecary-dithers) as the quest starter and ender for [The Key to Scholomance](https://tbc.wowhead.com/quest=5511/the-key-to-scholomance). Right now on AC it's another NPC, High Executor Derrington, who is not correct.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8551
- Closes https://github.com/chromiecraft/chromiecraft/issues/2039

## SOURCE:

https://wowpedia.fandom.com/wiki/The_Key_to_Scholomance?oldid=5155903
https://tbc.wowhead.com/quest=5511/the-key-to-scholomance

## Tests Performed:

- Ran SQL, no errors/warnings, 1 row changed per line as expected.
- Tested in-game:

Quest now offered by by Dithers:
![WoWScrnShot_101621_142048](https://user-images.githubusercontent.com/81782124/137573058-494a5320-b02b-4bcf-8e03-fcc699d24ec7.jpg)
![WoWScrnShot_101621_142051](https://user-images.githubusercontent.com/81782124/137573061-3a461012-add3-4836-a296-be67586c98b1.jpg)
![WoWScrnShot_101621_142054](https://user-images.githubusercontent.com/81782124/137573062-b4e63e87-a6c1-4a2b-91c6-c11cbc348289.jpg)

## How to Test the Changes:


1. .q add 5804
2. .additem [araj's scarab]
3. .go c 28415 and handin
4. Observe "The Key to Scholomance" is now offered.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
